### DISCUSSION TITLE
Option to select directory for standard template file

### DISCUSSION THREAD

**Discussion by burnuser** [freeplane/freeplane#435] [Ideas]
Beside standard templates (usually installed in an protected location, which needs admin rights for any change) every Map can also be saved as user template.
But in "Preferences - Environment - Files - Standard template file" only standard templates from the installation directory can be selected.

At least an option to select also user templates is needed, to complete the workflow with user templates.

And an additional free selection of the target directory would allow the usage of every Map at every place (e.g. easy manageable company templates from a network drive).

**Comment by dpolivaev** [comment]
Actually, it should have combined files from both directories with priority by the user directory file if the both exist

**Reply by macmarrum** [reply]
It does for me. I can choose a user template (located in my user templates directory) in `Preferences->Environment->Files->Standard template file`

**Comment by burnuser** [comment]
Yes, test with complete new map work as described. (Was not clear for me before.)

But the integration of a free addable (central) directory to the selection list would be fine too :-)
